Documentation/ceph-disaster-recovery.md

```diff
@@ -336,3 +336,17 @@ Assuming `dataHostPathData` is `/var/lib/rook`, and the `CephCluster` trying to
 1. Bring the Rook Ceph operator back online by running `kubectl -n rook-ceph edit deploy/rook-ceph-operator` and set `replicas` to `1`.
 1. Watch the operator logs with `kubectl -n rook-ceph logs -f rook-ceph-operator-xxxxxxx`, and wait until the orchestration has settled.
 1. **STATE**: Now the new cluster should be up and running with authentication enabled. `ceph -s` output should not change much comparing to previous steps.
+
```

Comment:
#4838 was written with the intention to restore a Rook cluster if everything was lost except the mon and osd PVCs, but there were a lot of open questions about how to do that. Perhaps it's better to abandon that PR and go with the assumption that you must back up the required resources if you expect to survive recovery. @tareqhs would you mind taking a look at that PR and see if there is anything useful to add here?

Reply:
is your intention to merge this into #4838 ?

Reply:
I went ahead and closed #4838 in favor of this one where we assume the metadata is backed up.

Reply:
thanks